src/bika/lims/content/report.py

```diff
@@ -41,9 +42,8 @@
             description=_("Report type"),
         ),
     ),
-    ReferenceField('Client',
+    UIDReferenceField('Client',
         allowed_types = ('Client',),
-        relationship = 'ReportClient',
         widget = ReferenceWidget(
```

Comment:
I think we need to take care here. Is the default `ReferenceWidget` coming from `Products.Archetypes` compatible with our `UIDReferenceField`?

Reply:
This `ReferenceWidget` is never rendered, the value for this field is always set programmatically here: https://github.com/senaite/senaite.core/blob/ea8189a638d20daedbbf6bea21f37851bf42b174/src/bika/lims/browser/reports/__init__.py#L301

However, will update the import to our `ReferenceWidget` in accordance (and make it `visible=False`)

Reply:
Done with https://github.com/senaite/senaite.core/pull/2221/commits/dc4f78ab2eed5d78b0febba038730a3a22ba3457